### ISSUE TITLE
CMS: Vet Center - Locations list table is missing caption

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#11162]
## Description
In the CMS, the Vet Center - Locations list displays "Main and Satellite Locations" in a table on the View screen.  However, this table uses a basic <div> element as a header for the table rather than a programmatic table caption, making it difficult for screen reader user to fully understand the exact purpose of the table date.

This View has also recently been update to include a second table displaying "Additional Mobile Vet Center Locations".  This table uses the same table as above, therefore it also is missing the associated table caption.

### Screenshot
![image](https://user-images.githubusercontent.com/106678594/194602336-3631118f-6466-4f02-86eb-4561575f5c70.png)

## Accessibility Criteria
[WCAG 1.3.1 A](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) 

## Acceptance Criteria
- [ ] Update existing "Main and Satellite Locations" table to use caption instead of div
- [ ] Update new "Additional Mobile Vet Center Locations" table to use caption instead of div

**Comment by omahane** [comment]
@laflannery please see the PR for these changes.
